internal/mode/static/state/change_processor.go

```diff
@@ -22,11 +22,6 @@ import (
 	"github.com/nginxinc/nginx-kubernetes-gateway/internal/mode/static/state/validation"
 )
 
-const (
```

Comment:
What do you think if we retain the context?  explaining why the validation failed in the first place.

the resource failed validation: Gateway API CEL validation (Kubernetes 1.25+) by the Kubernetes API server and the Gateway API webhook validation (if installed) failed to reject the resource with the error; make sure Gateway API CRDs include CEL validation and (if installed) the webhook is running correctly.